invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/bulk_actions/SelectedMembers.js

```diff
@@ -28,13 +28,25 @@ export class SelectedMembers extends Component {
         <Header as="h4" className="ml-20">
           {i18next.t(displayingGroups ? "Selected groups" : "Selected members")}
         </Header>
-        <Segment className="selected-members-header mb-20 mr-20 ml-20">
+        <Segment
+          className="selected-members-header mb-20 mr-20 ml-20"
+          role="list"
+        >
           {Object.entries(selectedMembers).map(([memberId, member]) => (
-            <Label className="mb-5 ml-5" image key={memberId}>
-              <Image src={member?.avatar} />
-              {member?.name}
-              <Icon onClick={() => this.removeMember(memberId)} name="delete" />
-            </Label>
+            <Button
+              className="p-0 ml-10"
+              onClick={() => this.removeMember(memberId)}
+              type="button"
+              role="listItem"
```

Comment:
The button should remain a button (not have the `role="listItem"` if it has button properties (i.e. it's clickable). Rather wrap the button in a div with `role="listItem"`. The aria-label should explain what is happening when you click the button. If you leave it a button, the tabindex is not needed.